cmd/benchmark.go

```diff
@@ -315,7 +320,8 @@ func rallyCommandAction(cmd *cobra.Command, args []string) error {
 		rally.WithProfile(profile),
 		rally.WithRallyTrackOutputDir(rallyTrackOutputDir),
 		rally.WithRallyDryRun(rallyDryRun),
-		rally.WithRallyPackageFromRegistry(packageFromRegistry),
+		rally.WithRallyPackageName(packageName),
+		rally.WithRallyPackageVersion(packageVersion),
```

Comment:
Nit. If they are always expected to be used together maybe we can add them as a single option.
```suggestion
		rally.WithRallyPackageFromRegistry(packageName, packageVersion),
```